.idea/shelf/Uncommitted_changes_before_Update_at_12-11-2021_16_31__Default_Changelist_.xml

```diff
@@ -0,0 +1,4 @@
+<changelist name="Uncommitted_changes_before_Update_at_12-11-2021_16_31_[Default_Changelist]" date="1636714895106" recycled="true" deleted="true">
+  <option name="PATH" value="$PROJECT_DIR$/.idea/shelf/Uncommitted_changes_before_Update_at_12-11-2021_16_31_[Default_Changelist]/shelved.patch" />
+  <option name="DESCRIPTION" value="Uncommitted changes before Update at 12-11-2021 16:31 [Default Changelist]" />
+</changelist>
```

Comment:
We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.